pkgs/applications/misc/athens/beta.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, fetchurl, appimageTools }:
+
+let
+  pname = "athens";
+  version = "2.0.0-beta.3";
+  name = "Athens-${version}";
```

Comment:
That is no longer really required since all the app image tools now understand pname+version, too.